pkg/secrethub/client.go

```diff
@@ -57,6 +57,16 @@ func NewClient(with ...ClientOption) (*Client, error) {
 			return nil, err
 		}
 	}
+
+	// Try to use default key credentials if none provided explicitly
+	if client.decrypter == nil {
```

Comment:
Only when the decrypter is `nil`? Or also when the `authenticator` is nil?

Reply:
Theoretically, yes. But... that is not accessible from the `secrethub` package. Since they should always be set at the same time, in practice the same effect is achieved. Not perfectly nice, but I'd say let's go with it for now and fix it if any problems arise from it.